packages/medusa/src/api/routes/store/auth/delete-session.ts

```diff
@@ -33,6 +33,11 @@
  *    $ref: "#/components/responses/500_error"
  */
 export default async (req, res) => {
-  req.session.jwt_store = {}
-  res.json({})
+  if(req.session.user_id) { // if we are also logged in as a user, persist that session
```

Comment:
todo: the commend seems a bit off

Reply:
The comment is correct

Reply:
the comments say that we are persisting the session where in fact we are just deleting a prop from the session that is why I was a bit confused by the comments